### DISCUSSION TITLE
Request a firmware build - 2.9.x

### DISCUSSION THREAD

**Discussion by pfeerick** [EdgeTX/edgetx#4000] [General]
Ask here if you want/need specific options that are not included in the stock builds. Builds will be uploaded to [here (same link as 'By request' below)](http://files.portabledev.xyz/d/96337458113243789080/), so check it out before making a request just in case there is already a custom build with your chosen options...

The Language builds are automated builds based on the 2.9 branch. 

*Please remember to specify your transmitter when requesting a build* ;)

- [Language builds](https://github.com/pfeerick/lang-firmwares/releases/tag/v2.9.0) - zip file per transmitter containing all languages built, with some additional builds for enabling bluetooth or lua mixer scripts for some radios. 
- [By request](http://files.portabledev.xyz/d/96337458113243789080/)

**Comment by D4RK3R2** [comment]
i need russian language  in tx12

**Reply by pfeerick** [reply]
As you may have realised by now this is not possible yet, but a PR to add `RU` support will be merged shortly, so it will be an option for 2.10.

**Comment by KevinFairgrieve** [comment]
Not sure this is possible!!

FrSky X12S BETA MACHINE. So it has the pre production motherboard, and no GPS. 

English please.

**Reply by pfeerick** [reply]
@3djc What flags do I need for the beta X12S?

**Reply by 3djc** [reply]
I think we removed support for those in otx before edgetx fork (since there had been a swap program by frsky) so my guess is you cannot

**Reply by KevinFairgrieve** [reply]
Thank you for looking at this, I suspected this was the case.

If it is possible in the future I would appreciate it.

**Comment by k120189** [comment]
could i get 
radiolink zorro with LUA mixers enabled?
English please

**Reply by pfeerick** [reply]
`FLAVOR=zorro EXTRA_OPTIONS="-DLUA_MIXER=Y" tools/build-gh.sh`

Available via the by requests link in the initial post.

**Comment by tulpek** [comment]
Can we have a build for Jumper T20 Hall Gimbals ELRS2.4G? Thank You

**Reply by pfeerick** [reply]
The Jumper T20 is not a supported radio for 2.9. Instead, you should be using the last factory firmware linked at https://edgetx.org/supportedradios/ or trying a nightly build of 2.10 if you're feeling adventurous.

**Comment by ilya-88** [comment]
Hello, can you help me assemble version 2.9 and 2 10 with External Access of the mod for the r9m long-range module from frsky for the radiomaster tx-16s. Thank you very much. when I assembled it didn’t work

**Reply by pfeerick** [reply]
Specific build for externalaccessmod is no longer necessary as of 2.9. If you have done the necessary hardware mod, instead set the AUX1 serial port to "External Module". https://manual.edgetx.org/color-radios/radio-settings/hardware#serial-port

**Comment by ilya-88** [comment]
Thank you!  I didn't know how it works, now I know!

**Comment by leonardnewnham** [comment]
Hi,
Could I please have a firmware build for an X9D Plus 2019 transmitter 
* with Lua enabled, and 
* without helicopter support
Many thanks!

**Comment by 3djc** [comment]
Lua is enabled by default, and heli can now be hidden by settings

**Reply by leonardnewnham** [reply]
OK, thank you, re heli.
Is Lua enabled by default for mixer scripts? I upgraded from OpenTX and could not run my scripts.

**Reply by 3djc** [reply]
Ok, so what you want is LUA_MIXER=Y

You can build it yourself from buddy using cloud build and adding the flag above
